src/executor/mod.rs

```diff
@@ -69,21 +85,29 @@ impl EvmExecutor {
                     .build();
                 cycle_tracker_end!("build Evm");
 
-                trace!("handler cfg: {:?}", revm.handler.cfg);
+                dev_trace!("handler cfg: {:?}", revm.handler.cfg);
 
                 cycle_tracker_start!("transact_commit");
-                let result = revm.transact_commit().unwrap(); // TODO: handle error
+                #[allow(unused_variables)]
```

Comment:
why is this required?

Reply:
Getting rid of unused_variables warning when we don't run it with feature-dev.